File: app/src/main/java/com/casebeaumonde/activities/register/userRegister/userRegisterResponse/UserRegisterResponse.java
package com.casebeaumonde.activities.register.userRegister.userRegisterResponse;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class UserRegisterResponse {

@SerializedName("code")
@Expose
private String code;
@SerializedName("message")
@Expose
private String message;

public String getCode() {
return code;
}

public void setCode(String code) {
this.code = code;
}

public String getMessage() {
return message;
}

public void setMessage(String message) {
this.message = message;
}

}

File: app/src/main/java/com/casebeaumonde/activities/userRegister/userRegisterResponse/UserRegisterResponse.kt
package com.casebeaumonde.activities.userRegister.userRegisterResponse

import com.google.gson.annotations.Expose

import com.google.gson.annotations.SerializedName


class UserRegisterResponse {
    @SerializedName("code")
    @Expose
    var code: Int? = null

    @SerializedName("message")
    @Expose
    var message: String? = null

}